src/psyclone/tests/domain/lfric/dyn_invoke_schedule_test.py

```diff
@@ -58,6 +58,6 @@ def test_dyninvsched_parent():
     dsched = DynInvokeSchedule("my_sched", kcalls)
     assert dsched.parent is None
     # With a parent
-    fake_parent = Container("my_mod")
+    fake_parent = Container("my_mod", symbol_table=LFRicSymbolTable())
```

Comment:
Do I understand this correctly that the only reason here is to get an LFRicSymbolTable (though no special functionality of the LFRic symbol table is used in this test)? It's good of course, I am just wondering if I am missing something :P

Reply:
Thanks for the explanation.